### DISCUSSION TITLE
HELP

### DISCUSSION THREAD

**Discussion by giugo17** [Sau1707/ModernBot#45] [General]
Sorry guys I download from the link but nothing works I just see the autoque once I open the Senate but no commands work! please help!

thanks in advance

**Comment by giugo17** [comment]
Maybe a guide for installation could help!

**Comment by Sau1707** [comment]
No need to download, install tampermonkey first, then it's installed directly

Here is the link for it https://www.tampermonkey.net/

**Reply by giugo17** [reply]
I have it already but does not work! would it help to disintsall and re-install everything? thanks for you anwer and help!

**Reply by Sau1707** [reply]
Do you see anything from the bot? Or it's not installed at all? If the bot is installed, open the console in the game with right click, "inspect", and the select "console" in the panel that has opened and send here the error

**Comment by drmacsofts** [comment]
Tampermonkey is installed and script is active but not respond in game,only
i see at senate building que to make order +- but press with mouse nothing
happen, also no other are available, no farm, no port, no soldier etc

Not working, check if is probably with new updates of game.

Στις Κυρ, 24 Μαρ 2024, 13:22 ο χρήστης Leonardo Saurwein <
***@***.***> έγραψε:

**Comment by drmacsofts** [comment]
As all we use it and respond that nothing work, what happened then? Is bot failure or inno check and block according new updates of game 2.308 is the last for me.

**Reply by Sau1707** [reply]
Open the console and see the error message there, I'm aware the is currently a bit broken, I'm working to fix it...it's just gonna take a bit, this is not my full time job

**Comment by FanSeaHUN** [comment]
Modern Farm doesn't work for me.
It counts to 0 and starts to count back agin, but but villages stay the same (no farm done)
Console:
![image](https://github.com/user-attachments/assets/1a9c8658-0c82-4e88-949c-bd4df0afba4f)

Tampermonkey:


![image](https://github.com/user-attachments/assets/7d0c2421-4713-4c65-9c3b-857e8bdabff0)

**Reply by FanSeaHUN** [reply]
Found this, maybe this is why the problem occurs:
![image](https://github.com/user-attachments/assets/8192140e-2cb6-4b1c-9bdd-ab22d684b0ab)
This is the 1.18.8 verion

In the 0.0.3 version it works if I keep the cursor idle

**Reply by PoLaKoSz** [reply]
- Just install [version 1](https://github.com/Sau1707/ModernBot/raw/refs/heads/main/dist/merged.user.js) then You should see an icon:
![image](https://github.com/user-attachments/assets/be031664-840b-43cf-b6c3-1b9e98c40df1)
- Click on it to start the auto farm (or just hover the mouse to see the options)

**Reply by FanSeaHUN** [reply]
The install was succesfull before, the icon that you mentioned shows up for me but doesn't work.
Doesn't matter if I set it for 5 min or 10 min, and set storage capacity to 100%, the number at the icon starts to count back, then after 0 it repeats itself with no farming done on the villages.
Everything else works in the script, but the FarmBot
![image](https://github.com/user-attachments/assets/add1c8e9-dd1e-4729-89d9-dec7e90bf9dd)

I noticed that if I turn off the GUI and the time's up, a lot of response error occurs in-game and I get theese errors:
![image](https://github.com/user-attachments/assets/ce2f063f-bc9b-4fb3-b3c8-414531ce4310)

Edit: I think I found out. I have to keep the game at the map, without anything open. Bot settings: 
![image](https://github.com/user-attachments/assets/6dbb7496-f9c6-482e-a744-be794dddc139)

**Comment by FanSeaHUN** [comment]
Is there an option to select what city recieves the farm from villages with the bot if there are multiple cities on one island?
I noticed that the bot goes on with the farming after the storage is already full despite I set the storage to 80% in the GUI
Also it's buggy when one city on the island only capable to receive the 5 min farm and the other can recieve 10min farm
I noticed that the bot doesn't collect anything from that island with this setup (2 city on 1 island, 1 is capable to collect 5 min other is 10), or only every 2-3nd time compared to other islands

**Comment by FanSeaHUN** [comment]
Hello!
Have you faced with an issue that if there are two cities on one island,
the bot stubbornly farms on one village no matter what?
I tried to reload the page, turn off then on the bot, disable the script
and re-enable it, but it's stuck farming that one city.
Even when that city is under conquest by an enemy player. It's rather not
farming on that village for the other city.

Tom PoLáKoSz ***@***.***> ezt írta (időpont: 2025. jan. 28.,
K, 6:05):
